jep/0000/README.adoc

```diff
@@ -0,0 +1,191 @@
+endif::[]
+
+.**JEP Template**
+[TIP]
+====
+In this document, all text in a "Tip" block (or inline text with with a ":bulb:" on either side)
+MUST be removed and/or replaced with text appropriate to this JEP before submission.
+
+Sections may include additional help and advice in comments.
+":bulb:" entries in comments only need to be filled in if that text is uncommented.
+
+See https://github.com/jenkinsci/jep/blob/master/jep/1/README.adoc[JEP-1] for full and accurate description of the JEP process and what is required in each section.
+====
+
+[TIP]
+====
+*BDFL-Delegate* is uncommented by default.
+As part of the in initial conversation or the JEP submission the sponsor should
+look for a BDFL Delegate.
+While not required, it is better for the community if Delegates perform most reviews.
+If no suitable BDFL-Delegate can be found, that row may be commented out.
+====
+
+.Metadata
+[cols="1h,1"]
+|===
+| JEP
+| 0000
+
+| Title
+| Continuous terminology update and identified offensive words including translations
+
+| Sponsor
+| :bulb: link:https://github.com/aHenryJard[aHenryJard], link:https://github.com/username2[User Name 2] :bulb:
+
+// Use the script `set-jep-status <jep-number> <status>` to update the status.
+| Status
+| Not Submitted :information_source:
+
+| Type
+| Informational
+
+| Created
+| Date (2021-06-22)
+
+| BDFL-Delegate
+| TBD
+
+
+| JIRA
+| https://issues.jenkins.io/browse/JENKINS-65398[JENKINS-65398 renaming Master to Controller/Built-in node ; whitelist and blacklist deprecation]
+
+| Discussions-To
+| https://community.jenkins.io/t/jenkins-terminology-cleanup-initiative-coordination/180
+
+
+|===
+
+== Abstract
+
+The goal of this JEP is to have the latest status of deprecated word or terms and agreed replacements in a table at one place that can be discussed with history.
+The proposal is to have git to keep history of changes, and discussion on new term on PRs.
+
+Deprecated term are written explicitly to be searchable.
+
+== Specification
+
+If you find a missing word in some language, please propose a PR on top of this JEP.
+
+=== English - Default
+
+* property suffix: <none>
+
+[options="header"]
+|======================
+| Deprecated term root           | New term root                 | New term declinaisons and comments               |
+| slave                          | agent                         | agents (plural)                                  |
+| master                         | controller                    | about the Jenkins instance                       |
+| master                         | built-in node                 | about the node embedded in the Jenkins instance  |
+| whitelist                      | allowlist                     | default but can be adapted to context            |
+| blacklist                      | denylist or blocklist         | default but can be adapted to context            |
+|======================
+
+=== Translations
+
+==== French - Français
```

Comment:
Could you please move all languages? Or do yio want to get format reviews first?

Reply:
done ;-) yes I was doing the formatting part with French only to have an easy copy/paste after